src/test/java/MainBannerTest.java

```diff
@@ -0,0 +1,48 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import pages.Homepage;
+
+import java.time.Duration;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class MainBannerTest {
+
+    private WebDriver driver;
+
+    @BeforeClass
+    public void setUp() {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/drivers/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.get("https://magento.softwaretestingboard.com");
+    }
+
+    @Test
+    public void homepage_verifyingCorrectTextContentOfMainBanner() {
+
+        //given
+        Homepage homepageA = new Homepage(driver);
+
+        //when
+        homepageA.getMainBannerTopText();
+        homepageA.getMainBannerMainText();
+        homepageA.getMainBannerButtonText();
```

Comment:
For what this 3 elements are taken here? It's redundant, as you do nothing with them. All 3 should be removed.